source/Calamari.AzureResourceGroup.Tests/AzureResourceGroupActionHandlerFixture.cs

```diff
@@ -69,37 +66,40 @@ public async Task Deploy_with_template_in_package()
-                                                     context.Variables.Add("Octopus.Action.Azure.TemplateSource", "Package");
-                                                     context.Variables.Add("Octopus.Action.Azure.ResourceGroupTemplate", "azure_website_template.json");
-                                                     context.Variables.Add("Octopus.Action.Azure.ResourceGroupTemplateParameters", "azure_website_params.json");
-                                                     context.WithFilesToCopy(packagePath);
-                                                 })
-                                    .Execute();
+                .WithArrange(context =>
+                {
+                    AddDefaults(context);
+                    context.Variables.Add(SpecialVariables.Action.Azure.ResourceGroupDeploymentMode, "Complete");
+                    context.Variables.Add("Octopus.Action.Azure.TemplateSource", "Package");
+                    context.Variables.Add("Octopus.Action.Azure.ResourceGroupTemplate", "azure_website_template.json");
+                    context.Variables.Add("Octopus.Action.Azure.ResourceGroupTemplateParameters", "azure_website_params.json");
+                    context.WithFilesToCopy(packagePath);
+                })
+                .Execute();
         }
 
         [Test]
         public async Task Deploy_with_template_inline()
         {
             var packagePath = TestEnvironment.GetTestPath("Packages", "AzureResourceGroup");
-            var paramsFileContent = File.ReadAllText(Path.Combine(packagePath, "azure_website_params.json"));
+            var templateFileContent = await File.ReadAllTextAsync(Path.Combine(packagePath, "azure_website_template.json"));
+            var paramsFileContent = await File.ReadAllTextAsync(Path.Combine(packagePath, "azure_website_params.json"));
             var parameters = JObject.Parse(paramsFileContent)["parameters"].ToString();
 
             await CommandTestBuilder.CreateAsync<DeployAzureResourceGroupCommand, Program>()
-                              .WithArrange(context =>
-                                           {
-                                               AddDefaults(context);
-                                               context.Variables.Add(SpecialVariables.Action.Azure.ResourceGroupDeploymentMode, "Complete");
-                                               context.Variables.Add("Octopus.Action.Azure.TemplateSource", "Inline");
-                                               context.Variables.Add(SpecialVariables.Action.Azure.ResourceGroupTemplate, File.ReadAllText(Path.Combine(packagePath, "azure_website_template.json")));
-                                               context.Variables.Add(SpecialVariables.Action.Azure.ResourceGroupTemplateParameters, parameters);
-
-                                               context.WithFilesToCopy(packagePath);
-                                           })
-                              .Execute();
```

Comment:
I applied "auto-indentation" for these code blocks, which makes the indentation pattern the same as that in Server. The old indentation pattern indents the parameter list of a method invocation to start from the same column. This pattern is hard to maintain. If we refactor the name of a method and the new name has a different length, then all the following lines of chained method invocation need to change the indentation accordingly to maintain a meaningful indentation style. This makes the diff trace unnecessarily large.